src/app/individual/individual-detail/individual-observation-view/individual-observation-view.component.ts

```diff
@@ -92,6 +100,19 @@ export class ObservationViewComponent implements OnInit {
         });
       })
     );
+    this.years$ = this.individual$.pipe(
+      switchMap(individual => this.individualService.getAllIndividualForAllYears(individual.individual)),
+      map(individuals =>
```

Comment:
This should not return the current year for public profiles if it has no observations. Also causes errors when displaying the page.

In general it would be safe to filter on individuals that have `last_observation_date` set (for public profiles). As quickfix we could also ng-if the html component and create a ticket to fix later.